packages/docs/pages/reference/elements/sharp.mdx

```diff
@@ -8,12 +8,18 @@ import {
 
 # `<sharp>`
 
-Short description of what the `<sharp>` tag does.
+The self-closing `<sharp/>` tag displays the sharp symbol in music notation. 
 
 ## Syntax
 
 <SyntaxDisplay name="sharp" />
 
 ## Examples
 
+```ptx-example
+<FRAGMENT parents="p">
+    In the context of music, the sharp symbol (<sharp/>) can be used in specifying key signatures or in modifying a single note.  It raises the pitch (of a note) by one semitone.  This contrasts with the flat symbol (<flat/>) which lowers the pitch by one semitone, thus
+    A<sharp/> has the same pitch as a B<flat/>.
```

Comment:
I believe the `<n>` tag should be used to write musical notes. Perhaps you can make reference to the programming language `F#`?

Also, please make sure the text in the examples does not exceed 80 characters per line. That way we can avoid scroll bars showing up.